packages/composite-checkout/src/components/checkout.js

```diff
@@ -96,32 +90,48 @@ const Container = styled.div`
 	}
 `;
 
-const Column = styled.div`
+const MainContent = styled.div`
 	background: ${props => props.theme.colors.surface};
-	padding: 16px;
 	width: 100%;
 	box-sizing: border-box;
+
 	@media ( ${props => props.theme.breakpoints.tabletUp} ) {
 		border: 1px solid ${props => props.theme.colors.borderColorLight};
-		margin-top: 32px;
+		margin: 32px auto;
 		box-sizing: border-box;
-		padding: 24px;
+		max-width: 556px;
 	}
 `;
 
-const MainContent = styled( Column )`
-	@media ( ${props => props.theme.breakpoints.tabletUp} ) {
-		max-width: 532px;
-	}
-`;
+function OrderSummaryStep( { CheckoutHeader } ) {
+	const localize = useLocalize();
 
-const PageTitle = styled.h1`
-	margin: 0;
-	font-weight: normal;
-	font-size: 24px;
-	color: ${props => props.theme.colors.textColorDark};
-	padding-bottom: 24px;
-`;
+	return (
+		<CheckoutStep
+			isActive={ false }
+			isComplete={ true }
+			stepNumber={ 0 }
+			title={ localize( 'You are all set to checkout' ) }
+			stepSummary={ <OrderSummaryContent CheckoutHeader={ CheckoutHeader } /> }
+		/>
+	);
+}
+
+OrderSummaryContent.propTypes = {
+	CheckoutHeader: PropTypes.elementType,
+};
+
+function OrderSummaryContent( { CheckoutHeader } ) {
+	return (
+		<React.Fragment>
+			{ CheckoutHeader ? <CheckoutHeader /> : <React.Fragment>Order Summary</React.Fragment> }
```

Comment:
"Order Summary" should be localized, and I wonder about this use of `Fragment`. 🤔 I get that you just want to return a text node, but something feels awkward to me about wrapping it in something. I think you can just change this whole component to:

```js
function OrderSummaryContent( { CheckoutHeader } ) {
  return CheckoutHeader ? <CheckoutHeader /> : localize( 'Order Summary' );
}
```

Reply:
Thanks for catching the localizing slip. I still run into some errors with this format, I'll just expand . it out for better readability.

Reply:
What do you think about it now @sirbrillig?